### ISSUE TITLE
Mapbox attribution

### ISSUE THREAD

**Issue by MikaKoskiola** [maplibre/maplibre-gl-native#79]
What are the Mapbox attribution requirements when using maplibre-gl-native on iOS & Android?

Mapbox stated the following (same applies for both Android and iOS): "You must include the Mapbox wordmark and attribution notice on any map that uses the Mapbox Maps SDK for Android".

Maplibre is still based on the Mapbox SDK which would suggest that attribution is required.

**Comment by wipfli** [comment]
Thanks for posting this here after opening the discussion in slack. In my opinion there is no requirement at all to put the logo or the name of Mapbox on your maps when using MapLibre.

**Comment by wipfli** [comment]
@nyurik @klokan @lseelenbinder what do you think?

**Comment by klokan** [comment]
Please provide the link / reference to where this statement appears.

By displaying maps from third party such as MapTiler.com, or from your own server, you are not using any Mapbox services nor Mapbox Maps SDK.

MapLibre Native and Android/iOS SDKs are powered by BSD licensed open-source code, where usage of the code has only the conditions specified in the license of the code. There is no proprietary code licensed under Mapbox Terms.

**Comment by lseelenbinder** [comment]
I agree with @klokan. I don't see this requirement for the BSD codebase (which this is based on).

**Comment by MikaKoskiola** [comment]
https://docs.mapbox.com/android/maps/guides/

"You must include the Mapbox wordmark and attribution notice on any map that uses the **Mapbox Maps SDK for Android**. "

This highlighted part has been there for a long time, as in years. IANAL, but when you have forked the Mapbox BSD licensed code base with this requirement as a starting point for Maplibre, you can't avoid this attribution requirement. As said, it's not about the map data or styles you use, but the SDK itself. I think you can't just go ahead and avoid original requirements by forking a given project.

**Comment by lseelenbinder** [comment]
Hi @MikaKoskiola,

Thanks for providing that link.

IANAL either, but in my reading of the linked page and the actual [LICENSE.md](https://github.com/maplibre/maplibre-gl-native/blob/master/LICENSE.md), it can't apply to simple code / built-code distributions (forked or not). The BSD license explicitly allows for redistribution and packaging with the only requirement being the inclusion of the BSD license and associated attribution. The only way additional requirements are added to that are via a separate agreement (such as agreeing to Mapbox's ToS), which using MapLibre GL Native doesn't require (unless you also consume Mapbox's services, which case you're required to use their SDK anyways).

A linked page can't remove explicitly granted rights in the code's license, so, of course, users must retain the included license (and make it available in the proper places), but I'm not sure how the linked requirement could be enforced. Additionally, Mapbox explicitly requested that we ensure there is no confusion between their SDKs and our forked projects, so requiring this would definitely blur that line.

I'm happy for more input here, but the BSD license is pretty clear, and I don't see how this page supersedes that given the text of LICENSE.md before the fork.

**Comment by matkoniecz** [comment]
> IANAL, but when you have forked the Mapbox BSD licensed code base with this requirement as a starting point for Maplibre, you can't avoid this attribution requirement.

Is it maube case similar to VSCode? Mapbox could impose some extra restrictions on their binaries and in their services, similarly how 

> Microsoft has released most of Visual Studio Code's source code on the microsoft/vscode repository of GitHub using the "Code – OSS" name, under the permissive MIT License,[5][10] while the releases by Microsoft are proprietary freeware.

https://en.wikipedia.org/wiki/Visual_Studio_Code
